bin/sequent

```diff
@@ -19,6 +19,9 @@ nextsteps = <<~NEXTSTEPS
     RACK_ENV=test bundle exec rake db:create db:migrate view_schema:build
     bundle exec rspec spec
 
+  To generate new aggregates use:
+    sequent generate <aggregate_name>. e.g. sequent generate address
```

Comment:
`sequent generate aggregate <aggregate_name>` would allow us to add `sequent generate event/command` later.

Reply:
Alias g = generate